util/state.js

```diff
@@ -347,6 +349,11 @@ return 0;
     return list.map(x => JSON.parse(x));
   }
 
+  async getErrorList() {
+    const list = await this.redis.hvals(this.ekey);
```

Comment:
This should be an `lrange` command since this is a list and not a hashmap:
```suggestion
    const list = await this.redis.lrange(this.ekey, 0, -1)
```